### PR TITLE
AB#2652 Add supported CP regions to Azure & GCP Docs

### PR DESCRIPTION
**Pull request by stdoutput**
### Proposed change(s)

- Adding an overview and / or tutorial on how to get the supported cloud provider regions to the documentation.

### Checklist

- [ ] Update [CHANGELOG.md](https://github.com/edgelesssys/constellation/blob/main/CHANGELOG.md)
- [ ] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [ ] Link to Milestone

### CODE REVIEW

File: docs/docs/getting-started/first-steps.md [hunk 1]
```diff
@@ -152,6 +159,8 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
     Fill the values produced by the script into your configuration file.
 
+    Please note that only regions which offer CVMs of the `N2D` series mentioned below are supported. Find a full list of regions [here](https://cloud.google.com/compute/docs/regions-zones). To see which regions offer the `N2D` series, filter by machine type: `N2D`. Alternatively, you can run `gcloud compute machine-types list --filter="name~n2d" --format="table[box,title='N2D series'](name,zone,description)"` to list all available machine types.
```
Comment (thomasten) [inline docs/docs/getting-started/first-steps.md:162]: Can you also add this hint to the "GCP (Console)" tab to the *regions* field? Maybe omit the gcloud command there.

File: docs/docs/getting-started/first-steps.md [hunk 2]
```diff
@@ -224,7 +233,7 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
     :::
 -->
 
-3. Create the cluster with one control-plane node and two worker nodes. `constellation create` uses options set in `constellation-conf.yaml`.
+1. Create the cluster with one control-plane node and two worker nodes. `constellation create` uses options set in `constellation-conf.yaml`.
```
[the remark this answers is not in the document]
Comment (datosh) [reply docs/docs/getting-started/first-steps.md:236]: 3. is correct and continues 2. from line 40. Content for cloud provider specific boxes is just extremely long 😉

File: docs/docs/getting-started/first-steps.md (hunk 1 above)
Comment (thomasten) [reply docs/docs/getting-started/first-steps.md]: Omitting the gcloud command was merely meant for the "GCP (Console)" tab as you don't interact with the shell there. I'd be fine with including it in the "GCP (CLI)" tab. I'm also fine with omitting it. Up to you.

File: docs/docs/getting-started/first-steps.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (thomasten) [reply docs/docs/getting-started/first-steps.md:236]: I think you need to revert this.

[the remark this answers is not in the document]
Comment (stdoutput) [reply docs/docs/getting-started/first-steps.md]: What do you mean by "revert" exactly?

[the remark this answers is not in the document]
Comment (datosh) [reply docs/docs/getting-started/first-steps.md]: Yes, latest version looks good to me!

**Review by datosh — APPROVED** [review]

**Review by thomasten — APPROVED** [review]

**Pull request merged**